WPMedia/fusion-news-theme-blocks
# PEN-1488 implemented Queryly integration

## Description
implemented Queryly integration

## Jira Ticket
- [PEN-1488](https://arcpublishing.atlassian.net/browse/PEN-1488)

## Acceptance Criteria
- A new Theme Setting called querylyId is created
- A new Theme Setting called querylyOrg is created
- If querylyId is present, add the following code to the end of the </body> on all pages across the website (per instructions in Queryly Instant Search doc):
  - `<script src="https://www.queryly.com/js/queryly.v4.js"></script> <script>     queryly.init('querylyId', document.querySelectorAll('main, .header-fix-nav')); </script>`
  - **Note**: The customer’s value for querylyId should be subbed in as shown above
- If a page has a page-type value of queryly-search, then the following script should be added below the scripts from AC # 2:
  - `<script src="https://www.queryly.com/js/querylyOrg-advanced-search.js"></script>`
  - **Note**: The customer’s value for querylyOrg should be subbed in as shown above

## Test Steps
- add the `querylyId` and `querylyOrg` to a site's properties
- check on that site the first two scripts are inserted
- create a new page and ad a metaTag value of `page-type: queryly-search`
- verify the 3rd script is inserted too

## Effect Of Changes
### Before
new functionality

### After
<img width="742" alt=" 2020-11-18-17 32 03" src="https://user-images.githubusercontent.com/9757/99590662-69cc5c80-29cc-11eb-8433-f206539c7efb.png">
<img width="1587" alt=" 2020-11-18-17 43 14" src="https://user-images.githubusercontent.com/9757/99590671-6cc74d00-29cc-11eb-834f-a33fd8d96c50.png">

## Dependencies or Side Effects
- two new properties on FP block.json file
  - `querylyId: 794b8320c4a14952` 
  - `querylyOrg: liberation`

## Review Checklist
please explain why (so that we can fix it whenever it gets refactored).
